build.xml

```diff
@@ -2223,6 +2223,7 @@
           <formatter type="plain"/>
           <formatter type="xml" tofile="${checkstyle.report.file}"/>
           <fileset dir="${build.src.java}" includes="**/*.java"/>
+          <!--<fileset dir="${test.dir}" includes="**/*.java"/>-->
```

Comment:
Left-over?

Reply:
Left it intentionally as there are two more tickets on the same line (thought it might be useful). I can remove, if you find it not useful.

Reply:
I was going to remove it on commit, no worries.